src/garage/sampler/worker.py

```diff
@@ -141,13 +143,18 @@ def update_agent(self, agent_update):
         """Update an agent, assuming it implements garage.Policy.
 
         Args:
-            agent_update(Dict[str, np.array]): Parameters to agent, which
-                should have been generated by calling
-                `policy.get_param_values`. Note that other implementations of
-                `Worker` may take different types for this parameter.
+            agent_update (np.ndarray or dict or garage.Policy): If a
```

Comment:
can we make this just like `env_update`? basically just a callable that's applied to the policy.

Reply:
can you leave an issue about making a type for parameters so that this interface can be less messy?